code/game/objects/structures/survey_probe.dm

```diff
@@ -0,0 +1,125 @@
+/obj/structure/survey_probe
+	name = "survey probe"
+	desc = "\
+		All-in-one survey probe, able to provide preliminary analysis of planetary bodies. \
+		It has different devices, samplers, and drill bits, as well as internal processing computers, \
+		to inspect the atmosphere, ground, soil, crust, and many other properties and qualities of planetary bodies. \
+		Commonly used by surveyors, explorers, pioneers, all over the Spur, looking for planets that are actually worth settling or exploiting for resources. \
+		This probe was manufactured by Orion Express, but it is based on on older model designed by Hephaestus Industries.\
+		"
+	desc_info = "\
+		The probe has to be deployed first before it is used. Wrench it to deploy, then click with empty hand to activate.\
+		"
+	icon = 'icons/obj/structure/survey_probe.dmi'
+	icon_state = "surveying_probe"
+	density = TRUE
+	/// If false, probe is not deployed.
+	/// If true, it is deployed and ready to survey.
+	anchored = FALSE
+	/// Timer of the survey process.
+	/// If null, it is not currently surveying.
+	var/timer_id = null
+
+/obj/structure/survey_probe/attackby(obj/item/item, mob/living/user)
+	if(!timer_id)
```

Comment:
Flip this to a guard or combine the two conditions

Reply:
done